app/routes/models/calculation.js

```diff
@@ -78,8 +107,6 @@ module.exports = function ViewModel (bpsValue, calculations) {
     paymentBand: createTableDefinition(calculations, 'rate', '', 'Payment band', 'percentage', false),
```

Comment:
`createTableDefinition` has a lot of parameters.  A better approach might be to have maybe two parameters, one for the table data and another for options.

For example:
```
function createTableDefinition (calculations, options) {
...
}
```
then use call it like:

```
const options = { 
  property: x,
  text: y,
  caption: z,
  formatType: a,
  showOverall: b
}
...
paymentBand: createTableDefinition(calculations, options)
```

or alternatively:

```
paymentBand: createTableDefinition(calculations, { property: x, text: y ... })
```